gradio/blocks.py

```diff
@@ -376,7 +376,7 @@ def __init__(
         self.preprocess = preprocess
         self.postprocess = postprocess
         self.tracks_progress = tracks_progress
-        self.concurrency_limit = concurrency_limit
+        self.concurrency_limit: int | None | Literal["default"] = concurrency_limit
```

Comment:
just wondering why you added the type hint here? its already specified in the method signature

Reply:
was getting type errors elsewhere because `self.concurrency_limit` was being treated as `int | None | str` otherwise

Reply:
Yea pyright doesn't seem remember about Literals if you access a property from a method